go_attack_utils/src/sgf_parser/game_info.py

```diff
@@ -132,16 +140,21 @@ def parse_game_str_to_dict(
     komi = float(komi) if komi else komi
     win_color = result[0].lower() if result else None
     assert (
-        "victim" in b_name or "victim" in w_name or "adv" in b_name or "adv" in w_name
+        "victim" in b_name
+        or "victim" in w_name
+        or "adv" in b_name
+        or "adv" in w_name
+        or victim_color
     ), f"Game doesn't have victim: path={path}, line_number={line_number}"
 
     parts = pathlib.Path(path).parts
     training = "eval" if "eval" in parts else "train" if "selfplay" in parts else None
 
-    victim_color = {b_name: "b", w_name: "w"}.get(
-        "victim",
-        "b" if ("victim" in b_name or "bot" in b_name or "adv" in w_name) else "w",
-    )
+    if victim_color is None:
+        victim_color = {b_name: "b", w_name: "w"}.get(
+            "victim",
+            "b" if ("victim" in b_name or "bot" in b_name or "adv" in w_name) else "w",
+        )
```

Comment:
the baseline attack script generates SGFs with the non-descript names `PB[black]PW[white]`, so I added the `victim_color` argument as an alternative way to specify which player is which

Reply:
If easy we might want to change baseline attack script to give more informative names, but this seems like a perfectly fine workaround for now.